console/account/how-to/resolve-abuse.mdx

```diff
@@ -0,0 +1,62 @@
+---
+meta:
+  title: How to resolve abuse tickets and cases
+  description: This page explains how to resolve abuse tickets.
+content:
+  h1: How to resolve abuse tickets and cases
+  paragraph: This page explains how to resolve abuse tickets.
+tags: resolve abuse complaint ticket scaleway
+dates:
+  validation: 2024-02-19
+  posted: 2024-02-19
+categories:
+  - account
+---
+
+Abuses are complaints regarding the use of Scaleway services for illegal behavior or hosting of illegal content. Abuse reports for the same issue on the same resource within a given timeframe are grouped by case.
+Scaleway’s clients are responsible for taking the appropriate security measures in order to solve the case and prevent any further abuse from their resources in order to avoid the locking of their services.
```

Comment:
```suggestion
If you have been the subject of an abuse report, you are responsible for taking the appropriate security measures to solve the case. You must also prevent any further abuse from your resources to avoid your services being locked by Scaleway.
```